src/JsonBuilder.cpp

```diff
@@ -1,10 +1,11 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-#include <cassert>
-
 #include <jsonbuilder/JsonBuilder.h>
```

Comment:
Note: Putting the `JsonBuilder.h` header first helps verify that the header isn't missing any dependencies.